src/runner/runner.hpp

```diff
@@ -46,12 +46,18 @@ namespace runner
 class Runner
 {
 public:
-  /// Instantiate all nodes without activating them.
+  /// Instantiate all nodes from parameter without activating them.
```

Comment:
Let's remove the default constructor. We only need one.

Reply:
That makes sense, let's implement `Runner()` using `Runner(std::vector<...>)` to avoid duplicating the code.